src/librbd/deep_copy/ObjectCopyRequest.h

```diff
@@ -135,6 +135,9 @@ class ObjectCopyRequest {
   void send_update_object_map();
   void handle_update_object_map(int r);
 
+  void trigger_copyup();
```

Comment:
You also need to update the state diagram above.

Reply:
Done